### DISCUSSION TITLE
Train without eos token?

### DISCUSSION THREAD

**Discussion by StableFluffy** [axolotl-ai-cloud/axolotl#990] [Q&A] [answered]
How to train with ignore eos token?

**Comment by NanoCode012** [comment]
Could you elaborate?

**Reply by StableFluffy** [reply]
I am trying to train model without eos token attatched.

**Reply by NanoCode012** [reply, marked as answer]
Hey, this would be an edge case. In such case, we support Bringing your own tokenized dataset. Please check readme but in essence, you’ll need to create your own prompt then tokenize it, and save as Dataset.

Afterward, you can pass the dataset path.
